### ISSUE TITLE
Cannot install Monterey in a z590 Vision D with a 11900K

### ISSUE THREAD

**Issue by GuitarFa** [SchmockLord/Gigabyte-Z590i-Vision-D-11900k#43]
Hi!
My name is Fernando and I have a Gigabyte z590 Vision D with a 11900K, but I can't install Monterey. When I changed my BIOS parameters (F8b BIOS version), I reboot, and then I choose the USB with your EFI. After some seconds, the screen change with an image of a mouse and then with a keyboard showing how to turn on, but my mouse and keyboard are connected by USB and don't work, also the keyboard don't turn on any LED.

Any idea of what happens with the USBs?

Thanks in advanced

**Comment by HoangYa** [comment]
put usb dongle or keyboard to USB with BIOS name above. Or change another USB port
i have same problem, and i put mouse dongle to another usb port, it's work

**Comment by SchmockLord** [comment]
@GuitarFa Yeah, it is because the Vision D needs a different USB Port config and you probably put it in these ports with no USB2.0 support.

Because of you I have added a Z590 Vision D USB Port config. You just have to disable SSDT-USB-Ports-Z590i-VisionD.aml and enable SSDT-USB-Ports-Z590-VisionD.aml.

**Comment by Crocdil** [comment]
Thanks for attaching a USB port configuration for the Z590-VisionD. Unfortunately it does not work for me at all. After I disabled (deleted) the configuration for the Z590i-VisionD and enabled the configuration for the Z590-VisionD, all USB ports are inactive. I have looked through the aml but I have not noticed any error. 

Only I can not find SSDT xh_rksu4 in my configuration that you disabled in config.plist. 

Thanks for the help.

**Comment by GuitarFa** [comment]
It works perfectly!! Thank you so much SchmockLord!!

**Comment by GuitarFa** [comment]
It works perfectly!! Thank you so much SchmockLord!!

**Comment by GuitarFa** [comment]
Hi again! Only two things:

Firstly, in about this Mac, my CPU appears like an I7-11900K. It's OK?

Secondly, when I work with audio I have some loops of 2 seconds when I'm  playing music and I don't know if it is because of the audio processing, the memory or because of these errors that I have in the config.plist file that I have seen with the OCAuxiliarityTools program:

OCS: No schema for PointerPollMask at 5 index, context <AppleInput>!
OCS: No schema for PointerPollMin at 6 index, context <AppleInput>!
OCS: No schema for PointerPollMax at 7 index, context <AppleInput>!
OCS: No schema for AudioOutMask at 2 index, context <Audio>!
OCS: No schema for DisconnectHda at 4 index, context <Audio>!
OCS: No schema for MaximumGain at 5 index, context <Audio>!
OCS: No schema for MinimumAssistGain at 6 index, context <Audio>!
OCS: No schema for MinimumAudibleGain at 7 index, context <Audio>!
OCS: Missing key AudioOut, context <Audio>!
OCS: Missing key MinimumVolume, context <Audio>!
OCS: Missing key VolumeAmplifier, context <Audio>!

Can you tell me how can I solve this errors? 

Thank you!

**Comment by HoangYa** [comment]
> 

Same problem
I'm using i7-11700K but About Us say i7-11900K :D 
But Sensei app show exactly i7-11700K
With version v31 i see ProcessorType is 0
And version V32 EFI show ProcessorType is 2431
So i think this number is problem with CPU name

**Comment by SchmockLord** [comment]
It is just cosmetic. If you want you can delete those entries
<img width="711" alt="Screenshot 2023-01-13 at 01 14 39" src="https://user-images.githubusercontent.com/19785918/212207973-d2224b81-2ca6-4030-9e96-427698979827.png">

**Comment by klougithub** [comment]
Can it work without problems also on a similar configuration but with a 10850k ?
I've the Z590 Vision D + 10850k + 6900XT + original WiFi Apple card on Pci-e 

I only have problems with USB ports and sometimes with sleep. 
So I'd love to try this EFI
(I didn't know it was suitable also for the ATX version)

**Comment by klougithub** [comment]
> It is just cosmetic. If you want you can delete those entries <img alt="Screenshot 2023-01-13 at 01 14 39" width="711" src="https://user-images.githubusercontent.com/19785918/212207973-d2224b81-2ca6-4030-9e96-427698979827.png">
> 
> And set ProcessorType back to 0.
> 
> Will do that with the next release.
DO you think it will work without problems also on a similar configuration but with a 10850k ?
I've the Z590 Vision D + 10850k + 6900XT + original WiFi Apple card on Pci-e

I only have problems with USB ports and sometimes with sleep.
So I'd love to try this EFI (I don't mind about integrated GPU)
(I didn't know it was suitable also for the ATX version)

Thanks for the great work and effort
